veml6070/veml6070.go

```diff
@@ -0,0 +1,138 @@
+// Package bmp180 provides a driver for the BMP180 digital pressure sensor
+// by Bosch.
+//
+// Datasheet:
+// https://www.vishay.com/docs/84277/veml6070.pdf
+// Application Notes:
+// https://www.vishay.com/docs/84310/designingveml6070.pdf
+//
+package veml6070 // import "tinygo.org/x/drivers/veml6070"
+
+import (
+	"time"
+
+	"machine"
+)
+
+// Device wraps an I2C connection to a VEML6070 device.
+type Device struct {
```

Comment:
We usually allow the I²C address to be configurable (for example https://github.com/tinygo-org/drivers/blob/master/bh1750/bh1750.go#L20 ), but this case is a bit special, as it uses two address for reading. Maybe it's a good idea to allow both to be configurable, or maybe configure one and let the other be +1. What do @aykevl & @deadprogram  think?

Reply:
I also thought about this but then this sensor only allows those two fixed addresses to be used - should it still be configurable?

Reply:
I think we need to expose the same interface, but what the driver actually does can be specific.

One reason we have the `Address` able to be changed, is to handle "clone" or "off-market" sensors that use different addresses. Another is the case that someone uses an I2C multiplexer like https://www.adafruit.com/product/2717

Basically, it seems best to allow it to be changed, even if the device itself does not.

Reply:
That's a good point, didn't thinkg about that... I now made both (the lower and the higher) addresses individually configurable so the user has maximum freedom - what do you think @deadprogram ?

Reply:
The change looks good to me.